runtime/nls/shrc/j9shr.nls

```diff
@@ -6877,3 +6877,11 @@ J9NLS_SHRC_SHRINIT_HELPTEXT_NO_TIMESTAMP_CHECKS_V1.system_action=
 J9NLS_SHRC_SHRINIT_HELPTEXT_NO_TIMESTAMP_CHECKS_V1.user_response=
 # END NON-TRANSLATABLE
 
+J9NLS_SHRC_CM_NEW_LAYER_CACHE_DESTROYED=The new layer of the cache is destroyed. Modification to a lower layer shared cache has been detected. Expected Cache Unique ID is %s, current Cache Unique ID is %s.
```

Comment:
I don't like the wording "The new layer". Please specify which layer of the cache was destroyed instead. Similarly, "to a lower layer" should instead specify which layer was modified.